### ISSUE TITLE
Multi-class Classification Raw Outputs

### ISSUE THREAD

**Issue by ns-hxue** [ThilinaRajapakse/simpletransformers#502]
I noticed the raw outputs from a multi class model has been changed since I updated to the new version. 

Assume I have total 6 classes, the raw model output has 10 arrays and each array has a length 6 elements. What does this mean? How to extract the score for this?

**Comment by ThilinaRajapakse** [comment]
I would guess that the 10 arrays correspond to the 10 samples, and the 6 elements correspond to the logits for the 6 classes.

**Comment by ns-hxue** [comment]
I don't think so. I only made prediction for one sample, and its raw
outputs return 10 arrays with 6 elements each.

**Comment by ns-hxue** [comment]
I have also noticed that it returns a different number of arrays for
different samples. What does this mean?

**Comment by ThilinaRajapakse** [comment]
Are you using the sliding window?

**Comment by ns-hxue** [comment]
Yes.

**Comment by ThilinaRajapakse** [comment]
Then the raw outputs should be the shape `num_samples * num_windows * num_classes`. The number of windows would vary based on the length of the sample in question.

Is this what you are getting or is the output messed up somehow?

**Comment by ns-hxue** [comment]
Oh, thanks for the clarification. To get the score for each sample, am i
supposed to get the mean of all arrays?

**Comment by ThilinaRajapakse** [comment]
Yes, that will give you the score for each sample.
